src/sqlfluff/rules/L019.py

```diff
@@ -134,7 +134,11 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
             # A comma preceded by a new line == a leading comma
             if context.segment.is_type("comma"):
                 last_seg = self._last_code_seg(context.raw_stack)
-                if last_seg and last_seg.is_type("newline"):
+                if (
+                    last_seg
+                    and last_seg.is_type("newline")
+                    and not last_seg.is_templated
```

Comment:
This line is the actual fix: Ignore templated newlines when looking for rule violations.